aws/table_aws_cloudwatch_metric_statistic_data_point.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	cloudwatchv1 "github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudWatchMetricStatisticDataPoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudwatch_metric_statistic_data_point",
+		Description: "AWS CloudWatch Metric Statistic Data Point",
+		List: &plugin.ListConfig{
+			Hydrate: listCloudWatchMetricStatisticDataPoints,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValue"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "metric_name",
+					Require: plugin.Required,
+				},
+				{
+					Name:    "namespace",
+					Require: plugin.Required,
+				},
+				{
+					Name:      "timestamp",
+					Operators: []string{">", ">=", "=", "<", "<="},
+					Require:   plugin.Required,
+				},
+				{
+					Name:    "period",
+					Require: plugin.Optional,
+				},
+				{
+					Name:    "dimensions",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(cloudwatchv1.EndpointsID),
+		Columns: awsRegionalColumns(cwMetricColumns([]*plugin.Column{
+			{
+				Name:        "label",
+				Description: "A label for the specified metric.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "period",
+				Description: "The granularity, in seconds, of the returned data points.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "dimensions",
+				Description: "The dimensions for the metric.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
```

Comment:
What are some example labels? Are they good titles?

Reply:
Labels are primarily based on resource id, and name or they can be the metric name as well. 

`i-0353536c53f7c8235 (test[sourav - test purpose] )`
`BucketSizeBytes`

Reply:
Are those helpful titles when looking at rows? From the examples above, it's not clear to me that they are helpful titles out of the box.